build/build.bash

```diff
@@ -1,6 +1,23 @@
 #!/bin/bash
 set -e
 
+rm -f /tmp/image
+
+if [ $HARVEY_GOLANG"" == "" ]; then
+               echo Not including Go in the image
```

Comment:
Strange tabbing. Once we have a build of go available we should probably make it non optional.

Reply:
we'll clean this up later I guess. I'd be way happier if this script were a Go program, I don't like building in bash dependencies.